pysat_requirements.txt

```
@@ -0,0 +1,8 @@
+dask
+netCDF4<1.5.7
+numpy<1.17
```

```suggestion
numpy
```
I don't think this is the issue